build/build-mrt.yml

```diff
@@ -93,12 +90,14 @@ steps:
     arguments: '${{ parameters.MRTSourcesDirectory }}'
     modifyEnvironment: true
 
+# TODO: Is this actually used? (ie: What installs it?)
 - task: powershell@2
-  displayName: 'Download .net runtime Windows installer'
+  displayName: 'Download .NET Runtime Windows installer'
   inputs:
     targetType: filePath
     workingDirectory: ${{ parameters.MRTSourcesDirectory }}\build
     filePath: ${{ parameters.MRTSourcesDirectory }}\build\DownloadDotNetRuntimeInstaller.ps1
+    arguments: -Platform $(buildPlatform) -OutputDirectory $(Build.SourcesDirectory)\BuildOutput\$(buildPlatform)\$(buildConfiguration)\TestDependencies
```

Comment:
>arguments: -Platform $(buildPlatform) -OutputDirectory $(Build.SourcesDirectory)\BuildOutput\$(buildPlatform)\$(buildConfiguration)\TestDependencies [](http://example.com/codeflow?start=4&length=148)

what is this for?

Reply:
I updated the DownloadDotNetRuntimeInstaller.ps1 script to take explicit paramters instead of inferring things from environment variables, so that I could share the script with the Helix job.